components/contracts/ActionContainer.tsx

```
@@ -93,98 +93,100 @@ const ActionContainer: FC<ActionContainerProps> = ({
+          {contract.status === 2 && !contract.feedbackFromInfluencer && (
+            <>
+              <Button
+                mode="contained"
+                style={{
+                  flex: 1,
+                }}
+                onPress={feedbackModalVisible}
+              >
+                Give Feedback
+              </Button>
+            </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)